Cleaned up the goals splash page (#26)
wayfair-incubator/auxeng-docs: src/docs/goals.md

featured: ./images/featured/goals.png
---

> ## AuxEng Program High Level Goals

Reviewer: ```suggestion
## AuxEng Program High-Level Goals
```

In addition to the primary goals, there are multiple secondary goals which are also valuable to organizations.

- It's an effective way of creating [mobility of engineering talent](mobility/) across the organization.
- It can provide an attractive [engineering role archetype](careers/) for the organization's most experienced software engineers.
- In addition to those, there are several other [organizational benefits](diffusion/) we've observed in the dozens of engagements we've run.

## Engagement Goals

Engagements can sometimes have different goals under different circumstances. Sometimes those goals will change how the engagement is run. For example, sometimes a shorter engagement of only a few weeks is sufficient to identify why a software team's commit size is increasing while their engineering cycle times are growing longer.

> We've used AuxEng to embed with an early adopter of a new tool to create a tight and clear feedback loop which expedites early iterations. The embedded engineer can see first hand how the tool is succeeding (or not) to achieve its own goals.

Reviewer: ```suggestion
We've used AuxEng to embed with an early adopter of a new tool, creating a tight and clear feedback loop which expedites early iterations. The embedded engineer can see firsthand how the tool is succeeding (or not) in achieving its own goals.
```

Here are a few example goals we've come across for specific engagements:

> - Drive Adoption of a tool or best practice

Reviewer: ```suggestion
- Drive adoption of a tool or best practice
```

> - Rally around a key business strategic goal without minimal disruption to other workstreams

Reviewer: You can fix this spellcheck warning by adding `workstreams` to `words/software.txt`

> - Rally around a key business strategic goal without minimal disruption to other workstreams

Reviewer: ```suggestion
- Rally around a key business strategic goal with minimal disruption to other workstreams
```

- Partner with an early adopter to understand how a new solution is being used
- Debug a behavioral or tooling issue that has been identified

> - Spread knowledge and awareness of a new technology or skillset

Reviewer: Same fix as [above](https://github.com/wayfair-incubator/auxeng-docs/pull/26/files#r934931262) for `skillset` (or you can just add a space to silence the spellcheck!)
